src/twisted/python/threadpool.py

```diff
@@ -43,7 +43,7 @@ class ThreadPool:
     name = None
 
     threadFactory = Thread
-    currentThread = staticmethod(currentThread)
+    current_thread = staticmethod(current_thread)
```

Comment:
This looks like externally-visible API.  Does this need to keep the old `currentThread` name around, perhaps with a Twisted deprecation notice?

Reply:
Of course this has to be even harder...  `:]`  Thanks for the review.  I agree.

Reply:
Why change the name in the Twisted API at all? An underscore name here is inconsistent with the rest of the class.